### PR TITLE
QA8-1396 Add function for test TC-102, add function to file methods.ja

### PR DESCRIPTION
**Pull request by Alex-Mashchenko**
Check my new code.

### CODE REVIEW

File: data/expected.json [hunk 1]
```diff
@@ -29,4 +29,4 @@
   "keyUp" : "123457",
   "keyDown" : "123455",
   "imageLabelSquare": "Click or drag and drop"
-}
+}
```
Comment (denispalkin) [inline data/expected.json:32]: Please add a new line after 32.

File: data/selectors.js [hunk 2]
```diff
@@ -21,4 +21,4 @@ const elements = {
   "imageSquare": ".ant-upload-select-picture-card",
 };
 
-module.exports = elements;
+module.exports = elements;
```
Comment (denispalkin) [inline data/selectors.js:24]: Add a new line after 24.

File: data/testData.js [hunk 3]
```diff
@@ -50,4 +50,4 @@ const story = {
   "comedy": 6,
 };
 
-module.exports = {name, gender, age, story};
+module.exports = {name, gender, age, story};
```
Comment (denispalkin) [inline data/testData.js:53]: Add a new line after 53.

File: helpers/methods.js [hunk 4]
```diff
@@ -1,11 +1,23 @@
...
+}
+
+function inputValuesExcStory(name, gender, age) {
+  $(sel.name).waitForDisplayed();
+  $(sel.name).setValue(name);
+  $$(sel.radioButtons)[gender].click();
+  $(sel.age).setValue(age);
+}
+
+module.exports = {inputValues4, inputStoryType7, inputValuesExcStory}
```
Comment (denispalkin) [inline helpers/methods.js:23]: Add a new line after 23.

File: test/extended/story.js [hunk 5]
```diff
@@ -1,121 +1,96 @@
...
-            expect(submitButton).toEqual(false);
-        });
+    it('TC-102 Type is not selected', () => {
+      inputValuesExcStory(name.default, gender.it, age.default)
+      let submitButton = $(sel.submit).isClickable();
+      expect(submitButton).toEqual(false);
     });
-});
+  });
+});
```
Comment (Mykhaylyak) [inline test/extended/story.js:96]: Add a new line after 96 line

File: test/smoke/functional.js [hunk 6]
```diff
@@ -23,7 +23,5 @@ describe('SMOKE TESTS SUITE', function () {
       let tryAgainBtn = $(sel.tryAgain).isDisplayed();
       expect(tryAgainBtn).toEqual(true);
     });
-
   });
-
-});
+});
```
Comment (Mykhaylyak) [inline test/smoke/functional.js:27]: Add a new line after 27 line

**Review by Mykhaylyak — CHANGES_REQUESTED** [review]

**Review by denispalkin — CHANGES_REQUESTED** [review]

File: test/extended/story.js (hunk 5 above)
Comment (Alex-Mashchenko) [reply test/extended/story.js]: Done

File: test/smoke/functional.js (hunk 6 above)
Comment (Alex-Mashchenko) [reply test/smoke/functional.js]: Done

File: data/expected.json (hunk 1 above)
Comment (Alex-Mashchenko) [reply data/expected.json]: Done

**Review by Mykhaylyak — APPROVED** [review]

**Review by denispalkin — APPROVED** [review]
Well done Jak!

**Review by Mykhaylyak — APPROVED** [review]

**Review by Mykhaylyak — APPROVED** [review]

**Review by Mykhaylyak — APPROVED** [review]

**Review by Mykhaylyak — APPROVED** [review]
Good job

**Review by denispalkin — APPROVED** [review]

**Pull request merged**